doc/configuration_options.rst

```diff
@@ -458,6 +458,11 @@ The following options are valid in version ``1`` (beside the generic options):
 * ``jenkins_job_upstream_triggers``: names of other CI jobs which, when
   built with a stable or unstable result, should trigger this job to be built.
 
+* ``jenkins_job_weight``: the number of executors on a worker which are
+  required to execute the job.
+  Default is ``1``.
+  Activates the Jenkins Heavy Job plugin.
```

Comment:
Since the plugin needs to be present anyway maybe rephrase this from "Activates ..." to just "Uses ..." or "Relies on ..."?

Reply:
2f0a74a